src/main/resources/templates/cla/ccla/sign.html

```diff
@@ -126,10 +126,12 @@
 				<button id="sign" type="submit" class="btn btn-success">Submit & Contribute</button>
 			</form>
 		</div>
+
 		<div layout:fragment="javascript">
+            <script th:src="@{/pages/cla/ccla/sign.js}" type="text/javascript"></script>
 			<script type="text/javascript">
-			$("[data-toggle=popover]").popover({html:true});
+				claCclaSign.initialize();
```

Comment:
Any chance we can externalize this too? I'd like to externalize everything so we can setup a Content Security Policy http://www.html5rocks.com/en/tutorials/security/content-security-policy/